tools/props/VersionsNet8OrLater.props

```diff
@@ -5,7 +5,6 @@
     <SystemConfigurationConfigurationManagerVersion>8.0.1</SystemConfigurationConfigurationManagerVersion>
     <MicrosoftExtensionsCachingMemoryVersion>8.0.1</MicrosoftExtensionsCachingMemoryVersion>
     <MicrosoftDotNetRemoteExecutorVersion>9.0.0-beta.24157.1</MicrosoftDotNetRemoteExecutorVersion>
-    <SystemTextJsonVersion>8.0.5</SystemTextJsonVersion>
```

Comment:
This is needed in order to override System.Text.Json included version with non-vulnerable version.